kuma/landing/jinja2/landing/homepage.html

```diff
@@ -122,8 +122,14 @@ <h2 class="offscreen">{{ _('Open Web and Mobile') }}</h2>
     <div class="column-callout callout-learn">
       <a href="{{ wiki_url('Learn') }}"><span>{{ _('Learn Web Development') }}</span></a>
     </div>
-    <div class="column-callout callout-deved">
-      <a href="https://www.mozilla.org/firefox/developer/?utm_source=developer.mozilla.org&utm_medium=referral&utm_campaign=mdn-front-pg-promo"><span>{{ _('Get The Browser For Developers Like You') }}</span></a>
+    <div class="column-callout callout-saucelabs">
+      <a href="https://saucelabs.com/cross-browser-testing-tutorial">
+        <span>
```

Comment:
This `<span>` keeps the text from overlapping the background image applied to the `<a>`.